preamble.tex

```diff
@@ -150,7 +150,12 @@
 \usepackage[nameinlink,noabbrev]{cleveref}
 \fi
 
+% There seems to be some major issue with using this as a package in LaTeXML messing up font in listing
```

Comment:
Could you add an example of a problem one could look for if investigating whether this is still needed in the future?

Reply:
That was the issue reported above on LaTeXML (with example).
I added a link to that.

Reply:
I guess the link to 1713 was added in the wrong place?  See suggestion above.

Reply:
No, this is also the right spot, there are two issues in 1713 - both importing this as package-issue and the new teletype-font.

Reply:
OK, I see.  Resolving.